src/ipahealthcheck/ipa/certs.py

```diff
@@ -188,6 +188,14 @@ def check(self):
                                      'nickname')
             notafter = request.prop_if.Get(certmonger.DBUS_CM_REQUEST_IF,
                                            'not-valid-after')
+            if notafter == 0:
+                yield Result(self, constants.WARNING,
```

Comment:
The assumption is that there is no real certificate issued, or at least none that certmonger was able to load. It is a message for the user to take a look what is going on w/the request.

To reproduce request a cert in a place SELinux won't allow. The tracking request will be created and certmonger will fail to get a cert b/c it can't generate the key, for example. There is still a request hanging around but not issued. The not-valid-after date will be retrieved as 0.

Reply:
Ack, so the problem I see is that if certmonger loses access to the directory for any reason, the request will not complete. Reporting this as constants.WARNING runs the risk of reporting a WARNING that would get ignored by well-meaning sysadmins until too late.

Apologies if the comment above was too terse, it really meant: "shouldn't this be constants.ERROR as there is no way the system would fix itself magically before an existing cert expires?"

Reply:
Sure, makes sense. Bumped message up to ERRORS.